libs/barista-components/radial-chart/src/radial-chart.ts

```diff
@@ -144,6 +155,10 @@ export class DtRadialChart implements AfterContentInit, OnDestroy {
   _selectable: boolean = false;
   static ngAcceptInputType_selectable: BooleanInput;
 
+  /* Notifies the component container of the start and end of hover events per series  */
+  @Output() hoverStart = new EventEmitter<DtChartHoverData>();
+  @Output() hoverEnd = new EventEmitter<DtChartHoverData>();
```

Comment:
Please provide one jsdoc comment per output - this will give the consumes proper help in their IDE.